css/screens/wishlist.css

```diff
@@ -0,0 +1,15 @@
+/* css for wishlist */
+
+
+
+.wishlist-contents{
+    flex-wrap: wrap;
+    padding: 2rem 0rem;
```

Comment:
it is better not to give units to 0 value, other than these everything looks good !